docs/resources/serving_endpoint.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "MLflow"
```

Comment:
Not sure if this is the best subcategory tbh. Could also belong under `Workspace` or something else?